src/utils/Storage.ts

```diff
@@ -0,0 +1,19 @@
+import { FileProps } from 'types/AppTypes'
+import localforage from 'localforage'
+import { Dispatch, SetStateAction } from 'react'
+import { FileObject } from './FileObject'
+
+export const GetStoredFiles = async (setFiles: Dispatch<SetStateAction<FileProps[]>>) => {
```

Comment:
```suggestion
export const getStoredFiles = async (setFiles: Dispatch<SetStateAction<FileProps[]>>) => {
```

funcões que não são componentes devem ser camelCase